lib/service/servicecfg/auth.go

```diff
@@ -253,6 +255,10 @@ func (cfg *PKCS11Config) CheckAndSetDefaults() error {
 	if cfg.SlotNumber == nil && cfg.TokenLabel == "" {
 		return trace.BadParameter("must provide one of SlotNumber or TokenLabel")
 	}
+
+	if cfg.MaxSessions == 1 {
+		return trace.BadParameter("the minimum value for PKCS11 MaxSessions is two")
```

Comment:
idk, the format we're actually looking for is a number
```suggestion
		return trace.BadParameter("the minimum value for PKCS11 MaxSessions is 2")
```